src/components/shared/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,40 @@
+import { Dispatch, PropsWithChildren, ReactNode, SetStateAction } from 'react'
+
+import styles from './Drawer.module.css'
+
+export default function Drawer({ isOpen, setIsOpen, boutonFermeture, children }: DrawerProps): ReactNode {
+  return (
+    <dialog
+      aria-modal="true"
+      className={`fr-modal ${styles['fr-modal']}`}
+      id="modal-drawer"
+      open={isOpen}
+    >
+      <div className="fr-grid-row fr-grid-row--right">
+        <div className="fr-col-4">
```

Comment:
```suggestion
        <div className="fr-col-12 fr-col-lg-4">
```
Pour avoir un effet plus sympa en mobile

Reply:
Vu avec David, pour l'affichage mobile, c'est plutôt la largeur minimale qui a été fixée à 320px.